src/App.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import './App.css';
+import BooksList from './container/BooksList';
+import BooksForm from './container/BooksForm';
 
 function App() {
   return (
-    <div className="App">
-      <h2>Bookstore Verissimo</h2>
+    <div>
+      <h2>BookStore</h2>
+      <table className="App">
+        <tbody>
+          <BooksList />
+        </tbody>
+      </table>
+      <BooksForm />
```

Comment:
- [ ] <b>REQUIRED:</b> By paying attention to Semantic HTML best practices, there is no point of using `table` here. instead, it will be needed in `BookList` component. Kindly consider refactoring.